### ISSUE TITLE
Support other ARM platforms

### ISSUE THREAD

**Issue by JJTech0130** [MinecraftMachina/multimc-macos-arm64#11]
I was just wondering if you were open to supporting builds for other ARM platforms, like the Raspberry Pi or Nintendo Switch. There is currently a build in pi-apps, but the maintainers of MultiMC don't seem to like it very much. You could ask @theofficialgman for more details about it.

**Comment by ViRb3** [comment]
Hey there! @theofficialgman and I have been in close contact since the birth of ManyMC, and I owe a lot of this project's success to him. I have raised the topic of merging his MultiMC fork into ManyMC before, and while we would both love to see a unified launcher for ARM, the answer is that it's a significant amount of work that neither of us is ready to take at the moment.

I don't plan to be the lead maintainer a Linux build for ManyMC since I don't use Linux for anything but a server. I can't offer the quality that I would expect if I don't even use the program. If another platform were to join ManyMC, it would need to come from a dedicated maintainer, I can only offer help with setting up a unified infrastructure. That said, there are some significant blockers such as the current manifest format (inherited from MultiMC) being completely unaware of different architectures for each platform.

To sum it up, this is definitely something I would want, but it is a lot of work that will take time.

**Comment by JJTech0130** [comment]
Ah, that's OK. I didn't realize @theofficialgman was already involved. I used to be able to sink large amounts of time into things like this (that's how I developed my initial tools for this), but I've become a part of several new things, and I don't really play Minecraft on a daily basis. (Honestly, I don't use my Pi for anything other that a server at this point...) I was originally thinking more along the lines of using your name (ManyMC) in the pi-apps version (to placate @peterix), but I see how that might give you unwanted workload and/or responsibility.

**EDIT**: As a side note, I just found out that you can edit in the @ in someone's name to link to their profile, hopefully without pinging them.
